### ISSUE TITLE
python file for generation line points and visualization

### ISSUE THREAD

**Issue by RyanHangZhou** [voldemortX/pytorch-auto-drive#73]
Hey, very impressive work on lane detection. I'm wondering if you could provide a simple file for easy generating line points and its visualization given an arbitrary street scene image? Thanks!

**Comment by voldemortX** [comment]
@RyanHangZhou Do you mean something like visualizing a line (defined by sample points), on a given image?

**Comment by voldemortX** [comment]
tools/vis/lane_img_dir.py can visualize CULane format line txts on a folder of images.

**Comment by RyanHangZhou** [comment]
> 

Yes, I find the configuration of datasets complex to me. I may not need configurating them for training. I'd like to use the model for providing points and its orientations with only a given image.

**Comment by RyanHangZhou** [comment]
> 



> tools/vis/lane_img_dir.py can visualize CULane format line txts on a folder of images.
I also encounter issues like this:

`  File "tools/vis/lane_img_dir.py", line 4, in <module> 
    from importmagician import import_from                 
ModuleNotFoundError: No module named 'importmagician'`

Didn't find this `importmagician` module in the directory ...

**Comment by voldemortX** [comment]
you'll need to install it via `pip install importmagician`

**Comment by voldemortX** [comment]
> > 
> 
> Yes, I find the configuration of datasets complex to me. I may not need configurating them for training. I'd like to use the model for providing points and its orientations with only a given image.

It is difficult to maintain minimum code for any one exact application, but the present vis code should be in relatively the easiest way, especially when you consider using a model to inference.
